### PR TITLE
feat: add TTL and max size for cache storages in proxy repos

### PR DESCRIPTION
**Pull request by genryxy**
Part of #334 
Added interface for obtaining TTL and max size values from config of repo. In next PR, I'm planning to add some tests

### CODE REVIEW

File: src/main/java/com/artipie/repo/YamlCacheStorage.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ * Proxy cache storage config from YAML.
+ * @since 0.23
+ */
+final class YamlCacheStorage implements CacheStorage {
+    /**
+     * Cache storage.
+     */
+    private final Storage cstorage;
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/repo/YamlCacheStorage.java:20]: I'd suggest to simplify field name to `asto`

File: src/main/java/com/artipie/repo/YamlCacheStorage.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    /**
+     * Max available size.
+     */
+    private final Long cmaxsize;
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/repo/YamlCacheStorage.java:25]: And here to `size`

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, please, take a look, I have naming suggestions

**Comment by genryxy** [comment]
@olenagerasimova thanks, renamed

**Comment by olenagerasimova** [comment]
@genryxy thanks, looks good to me
@g4s8 please, take a look/merge

**Comment by genryxy** [comment]
One integration test fails although changes are not related to this
```
[INFO] Running com.artipie.api.ArtipieApiITCase
artipie/artipie:1.0-SNAPSHOT - artipie: [INFO] 2021-09-01 08:27:10,824 main com.artipie.VertxMain - Used version of Artipie: 1.0-SNAPSHOT
Error: /artipie:1.0-SNAPSHOT - artipie: [ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 2, Time elapsed: 3.808 s <<< FAILURE! - in com.artipie.api.ArtipieApiITCase
Error:  com.artipie.api.ArtipieApiITCase.dashboardIsUp  Time elapsed: 3.803 s  <<< FAILURE!
java.lang.AssertionError: 
Artipie dashboard is not up and running
Expected: status is <0>stdout a string containing "<!DOCTYPE html>", "<title>alice</title>", "Your repositories:" in order
     but: failed with status:
<7>
STDOUT: 
```

**Comment by olenagerasimova** [comment]
@genryxy ubuntu 11 log says it's ok
```
rtipie:1.0-SNAPSHOT - artipie: [WARNING] Tests run: 3, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 3.427 s - in com.artipie.api.ArtipieApiITCase
```

**Comment by genryxy** [comment]
@g4s8 could you please take a look

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Comment by g4s8** [comment]
@olenagerasimova please click approve if this PR is OK for you

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**